src/login/login_auth.cpp

```diff
@@ -107,7 +107,9 @@ int32 login_parse(int32 fd)
             const char* fmtQuery = "SELECT accounts.id,accounts.status \
 									FROM accounts \
 									WHERE accounts.login = '%s' AND accounts.password = PASSWORD('%s')";
-            int32 ret = Sql_Query(SqlHandle, fmtQuery, name.c_str(), password.c_str());
+            Sql_EscapeString(SqlHandle, chrName, chrName);
```

Comment:
Sql_EscapeString's comments: 
`/// The output buffer must be at least strlen(from)*2+1 in size.`
Because of this, it is probably not safe to use the same buffer as input and output.  We probably need to allocate a new buffer with the size of name.length()*2 +1 (and same for password)
we would probably want something like
`char[16*2+1] escaped_name` and so on

I don't have a compiler in github obviously, so just message me on discord if you have trouble getting that working

Reply:
:+1: